webpack_config/electron-main.development.js

```diff
@@ -114,7 +114,8 @@ module.exports = {
     new CleanWebpackPlugin(),
 
     new webpack.EnvironmentPlugin({
-      'NODE_ENV': 'development'
+      'NODE_ENV': 'development',
+      'TARGET_ENV': 'electron'
```

Comment:
Seems to be a duplicate definition with `dev:electron` command in `package.json`.
Use one or the other.